### ISSUE TITLE
HUD Lights

### ISSUE THREAD

**Issue by jwdunn1** [freshfork/p5.EasyCam#1]
If lights are **on**, then screen-aligned objects rendered in HUD mode can be affected by the orientation of the camera. For example, given a scene with lights() and a box, if the camera is rotated around the scene, a filled rectangle rendered in the HUD context will vary in brightness. The lights should probably be turned off automatically for the HUD rendering, perhaps using noLights() (when that is implemented) or by setting this._renderer._enableLighting = false;

**Comment by jwdunn1** [comment]
Work around example...
~~~js
  easycam.beginHUD();
    this._renderer._enableLighting = false; // fix for issue #1
    ...
  easycam.endHUD();
~~~

**Comment by jwdunn1** [comment]
noLights() is now implemented. I recommend calling noLights() immediately following beginHUD().
~~~js
  easycam.beginHUD();
    noLights();
    ...
  easycam.endHUD();
~~~
